packages/wonder-blocks-dropdown/src/components/combobox.tsx

```diff
@@ -753,15 +756,22 @@ const styles = StyleSheet.create({
         border: "none",
         outline: "none",
         padding: 0,
-        minWidth: spacing.xxxSmall_4,
-        width: "auto",
-        display: "inline-grid",
-        gridArea: "1 / 2",
```

Comment:
Moving this to the styles for the TextField root element. I moved it as is since I know others are also working in the dropdown package so I didn't want to make too many changes in here like updating to use `sizing` tokens!

Reply:
This is great!